data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
```

Comment:
I agree with changing the shields and hull.  Then double the base mass but leave all the space allocations alone.  This would make them lumbering giants, easy to avoid but extremely dangerous to close with.
Do remember that Mules and Bactrians  are relics of an older, far more dangerous time when the galaxy was sparsely settled. The time of the Alpha wars.  And they are designed to transport complete colonies.
Thus they need to be able to operate for extended periods without support against hostilities.
For their purpose they need to be able to take heavy punishment, give it as good (or better) as they get and carry huge loads at a high strategic speed.  They can however afford to have a low tactical speed,